src/de/read.rs

```diff
@@ -99,7 +171,7 @@ impl<R: io::Read> io::Read for IoReader<R> {
 }
 
 impl<'storage> SliceReader<'storage> {
-    #[inline(always)]
+    #[inline(never)]
```

Comment:
I'm not sure if this change is needed, could you do a benchmark with and without this change?